force-app/main/default/classes/StringGenerator.cls

```diff
@@ -0,0 +1,30 @@
+public with sharing class StringGenerator {
+
+    private List<String> my_list = new List<String>();
+
+    public StringGenerator( List<String> my_list_by_user )
+    {
+        my_list = my_list_by_user;
+    }
+    
+    public List<String> returnRandomString()
+    {
+        List<Integer> randomIndexList = new List<integer>();
```

Comment:
case for thr typr should always start with capital. Instead of new List<integer>(), it will be new List<Integer>()

Reply:
"Instead of new List(), it will be new List()"? Sorry, I didn't quite understand this point

Reply:
List<Integer> * dont know how it got missed